app/common/themes/HighContrastLight.ts

```diff
@@ -16,6 +16,7 @@ export const HighContrastLight: ThemeTokens = {
   primary: '#0f7b51',
   primaryMuted: '#196C47',
   primaryDim: '#196C47',
+  primaryTransparent: 'rgba(15, 123, 81, 0.5)',
```

Comment:
This is the old notation. The modern one would be:
```suggestion
primaryTransparent: 'rgb(15 123 81 / 0.5)',
```